doc/sphinxman/source/build_planning.rst

```diff
@@ -199,13 +199,13 @@ that software for |PSIfour| and any notes and warnings pertaining to it.
 * :ref:`Optimized BLAS and LAPACK libraries <cmake:lapack>` (preferably NOT one supplied by a standard
   Linux distribution)
 
-* :ref:`Python interpreter and headers <cmake:python>` (3.6 or 3.7) https://www.python.org/
+* :ref:`Python interpreter and headers <cmake:python>` (3.6+) https://www.python.org/
 
 * CMake (3.8+) https://cmake.org/download/
```

Comment:
The CMake requirement was bumped to 3.12 or 3.15.

Reply:
good point. that file needs more work.